lib/src/query_parameters/keys_query.dart

```diff
@@ -0,0 +1,16 @@
+class KeysQuery {
+  final int? offset;
+  final int? limit;
+
+  KeysQuery({
+    this.limit,
+    this.offset,
+  });
+
+  Map<String, dynamic> toQuery() {
+    return <String, dynamic>{
+      'offset': this.offset,
+      'limit': this.limit,
+    }..removeWhere((key, value) => value == null);
+  }
+}
```

Comment:
For the Dart, you prefer to create a class for each return. I think I prefer this way too!